Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool PAUSED = false;
```

Comment:
false initialization is not necessary, default value of boolean is false

Reply:
I would like to keep it showing as false, so that it is obvious that it starts as false.